salt/transactional-update/10-update-rebootmgr-options.conf

```diff
@@ -0,0 +1,3 @@
+[Service]
+ExecStart=
+ExecStart=/usr/sbin/transactional-update cleanup dup salt
```

Comment:
> For normal updates this might be fine, but there is some updates that will need much faster deployments. (libssl and similar)

Those should be treated as critical and 12 hours can also be a really bad window for something with such an impact.

Maybe we can skip calling `transactional-update` (thus, avoiding an extra snapshot) if we check on the systemd unit that the grain is already there. If it is, we skip calling transactional-update. This would skip intermediate updates since the last snapshot though.

Just wanted to point another possible approach, but I'm okay with your reasoning given where we are.